src/panels/lovelace/cards/hui-shopping-list-card.ts

```diff
@@ -120,28 +131,27 @@ class HuiShoppingListCard extends SubscribeMixin(LitElement)
             )}
             @keydown=${this._addKeyPress}
           ></paper-input>
+          <ha-icon
+            class="reorderButton"
+            icon="hass:sort"
+            .title=${this.hass!.localize(
+              "ui.panel.lovelace.cards.shopping-list.reorder_items"
+            )}
+            @click=${this._toggleReorder}
+          >
+          </ha-icon>
```

Comment:
ha-svg-icon

Reply:
I see that the whole file is still using `ha-icon`, I will migrate them all in this PR.